src/store/movie/operations/getMovieCreditsOperation.js

```diff
@@ -0,0 +1,34 @@
+import { createLogic } from 'redux-logic'
+import { normalize } from 'normalizr'
+import { uniq } from 'lodash'
+
+import { movieCredits } from 'Schemas'
+import { storeData } from 'Store/data/actions'
+import { movieSelector } from 'Store/movie/selectors'
+import { GET_MOVIE_CREDITS } from '../types'
+import {
+  requestMovieCreditsStart,
+  requestMovieCreditsSuccess,
+  requestMovieCreditsError
+} from '../actions'
+
+export const getMovieCreditsOperation = createLogic({
+  type: GET_MOVIE_CREDITS,
+  latest: true,
+  async process({ action, axios, getState }, dispatch, done) {
+    const { movieId } = action
+    dispatch(requestMovieCreditsStart())
+    try {
+      const response = await axios.get(`/movie/${movieId}/credits`)
+      const { movie, persons } = normalize(response.data, movieCredits).entities
+      const storeMovie = movieSelector(getState(), movieId)
+      const { cast, crew } = movie[movieId]
+      dispatch(storeData('persons', persons))
+      dispatch(storeData('movies', { [movieId]: { ...storeMovie, cast, crew: uniq(crew) } }))
```

Comment:
we don't need to merge fresh movie data with existed data in store here. data reducer should handle this

Reply:
@VladWithCookies in the movie object a store only cast & crew IDs, I have a separate data slice called `persons` where I store persons entities